tests/test_audit_paths_linmou.py

```
@@ -221,3 +221,92 @@ def test_inv5_red_when_one_intent_unreachable():
+    from collections import deque
+    nodes = tree["nodes"]
+    start = "n_l1985_entry"
+    seen = {start}
+    q = deque([start])
+    while q:
+        cur = q.popleft()
+        node = nodes.get(cur, {})
+        for ch in node.get("choices", []) or []:
+            nxt = ch.get("next")
+            if nxt and nxt not in seen and nxt in nodes:
+                seen.add(nxt); q.append(nxt)
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)